utils.js

```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const path = require('path');
 
 const glob = require('glob');
+const ignore = require('ignore');
```

Comment:
npm has a pkg specialised for gitignore/npmignore be easier to maintain?

https://github.com/npm/ignore-walk#ignore-walk

if it's easier to keep as is, no need

Reply:
Nice idea, but that returns with a list of all files, which we'd then have to parse the user-provided glob and filter that list down again.